
<launch> 
	  <rosparam command="load" ns="robot_description_planning" file="$(find pr2_arm_navigation_config)/config/pr2_planning_description.yaml" />

  <param name="robot_description_file_name" value="$(find planning_models)/test_urdf/robot.xml"/>
	<node pkg="chomp_motion_planner" name="chomp_motion_planner" type="chomp_planner_node" args="collision_map:=collision_map_occ" respawn="true" output="screen" clear_params="true" launch-prefix="xterm -e gdb --args">
                <rosparam command="load" file="$(find chomp_motion_planner)/config/chomp_pr2_right_arm.yaml" />
	
		<param name="animate_path" value="false"/>
		
		<param name="trajectory_duration" value="3.0"/>
		<param name="trajectory_discretization" value="0.02"/>
		<param name="learning_rate" value="100000.0" />
		<param name="max_iterations" value="500" />
		<param name="max_iterations_after_collision_free" value="100" />
		<param name="smoothness_cost_weight" value="0.0005"/>
		<param name="obstacle_cost_weight" value="1.0" />
		<param name="joint_update_limit" value="0.01" />

		<param name="smoothness_cost_velocity" value="0.01" />
		<param name="smoothness_cost_acceleration" value="1.0" />
		<param name="smoothness_cost_jerk" value="0.000001" />

		<param name="add_randomness" value="true" />
	</node>
</launch>
